### DISCUSSION TITLE
Default template for "Create: New Jupyter Notebook"

### DISCUSSION THREAD

**Discussion by vinaychuri** [microsoft/vscode-jupyter#13222] [Questions and Answers]
In VSCODE we have the option to create a new Jupyter notebook via Shift+Cmd+N or Shift+Cmd+P->"Create: New Jupyter Notebook"
This opens the editor to a new Jupyter notebook with an empty first cell.

Is there an option to edit this default new file behavior so users can load the first cell prefilled with imports, comments, configs, etc?

**Comment by Kaschi14** [comment]
I'd love to have this option as well :)
